packages/terra-site/package.json

```diff
@@ -110,6 +110,7 @@
     "style-loader": "^0.19.0",
     "terra-divider": "^1.0.0",
     "webpack": "^3.6.0",
-    "webpack-dev-server": "2.7.1"
+    "webpack-dev-server": "2.7.1",
+    "terra-spacer": "^0.0.0"
```

Comment:
Move terra-spacer to dependencies

Reply:
@emilyrohrbough After discussing this @bjankord we decided to keep this as dev dependency until it is released.

Reply:
Can you or @bjankord  explain what the problem is with listing unreleased components as dependencies? The `terra-divider` component was the first that we'd done that with. We have several unreleased components out for PR that are listed as dependencies in terra-site https://github.com/cerner/terra-core/pull/979, https://github.com/cerner/terra-core/pull/982, and https://github.com/cerner/terra-core/pull/978 so if there is a problem we would want to update those. I also noticed that `terra-divider` was released and didn't get changed over to be a dependency.

Reply:
The reason the components in terra-site should be a dependency is that we are publishing these examples component examples so we can reuse them in the terra-ui site. For example, we are pulling in this [terra-site button example](https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/button/ButtonSize.jsx) into terra-ui [here](https://github.com/cerner/terra-ui.com/blob/master/client/app/bundles/App/Components/Core/Button/Index.jsx#L10).

Reply:
@jmsv6d So the discussion I had with @bjankord was, when I listed terra-divider as dependency in package.json and do an npm install to update package-lock.json it was spitting out the error below.

lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm install
lerna ERR! npm ERR! code E404
lerna ERR! npm ERR! 404 Not Found: terra-divider@^1.0.0
lerna ERR!
lerna ERR! npm ERR! A complete log of this run can be found in:
lerna ERR! npm ERR!     /Users/sm054012/.npm/_logs/2017-11-14T15_52_19_394Z-debug.log
lerna ERR!
lerna ERR!     at Promise.all.then.arr (/Users/sm054012/Desktop/terra-core/node_modules/lerna/node_modules/execa/index.js:236:11)
lerna ERR!     at <anonymous>
(node:21690) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): Error: Command failed: npm install
npm ERR! code E404
npm ERR! 404 Not Found: terra-divider@^1.0.0

After investigating we found out that package.json was assuming terra-divider to be a npm package which it is not (before release). So until release we decided to make it as dev dependency and once it is published we wanted to make it a dependency. But now listing terra-spacer as dependency with version 0.0.0 (realized this later now) and doing a npm install, didn't spit out any error.

Reply:
It sounds like that was just a mistake of making the dependency for ^1.0.0 instead of ^0.0.0. You should move this to dependencies (and terra-divider while you're in there).

Reply:
@jmsv6d Updated https://github.com/cerner/terra-core/pull/1011/commits/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-2f4ced636a7ac23a1b1498c37d644ae1R75